styles/components/flexbox-hero.scss

```diff
@@ -0,0 +1,7 @@
+.flexbox-hero {
+  display: flex;
+  justify-content: center;
+  height: 40vh;
+  padding-top: 5rem;
```

Comment:
This padding top is not properly centering the flexbox hero contents. Instead look at removing the `padding-top` and using `align-items` instead.